app/models/webhook.rb

```diff
@@ -77,8 +88,8 @@ def rendered_payload(event_name, payload)
     webhook_template.render(
       variables: {
         event_name: event_name,
-        object: payload.delete(:object),
-        context: payload.delete(:context),
+        object: payload[:object],
+        context: payload[:context],
         payload: payload,
         webhook_id: id
```

Comment:
This was hard to track down as I was testing two webhooks for a same event. These kind of things are super dangerous in any language which uses references for arguments (most of them).